### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by robksawyer** [leny/atom-coffeescript-check#4]
[Enter steps to reproduce below:]

1. Pressed Cmd + P 
2. Navigated to Check Coffeescript, pressed Enter.

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [coffeescript-check](https://github.com/leny/atom-coffeescript-check) package, v0.1.0

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/robsawyer/.atom/packages/coffeescript-check/lib/coffeescript-check.coffee:16

TypeError: undefined is not a function
  at coffeeCheck (/Users/robsawyer/.atom/packages/coffeescript-check/lib/coffeescript-check.coffee:16:17)
  at atom-text-editor.<anonymous> (/Users/robsawyer/.atom/packages/coffeescript-check/lib/coffeescript-check.coffee:38:13)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -0:57 core:copy (atom-text-editor.editor.is-focused)
     -0:52.3.0 editor:newline (atom-text-editor.editor.is-focused)
  6x -0:52 core:backspace (atom-text-editor.editor.is-focused)
     -0:50.1.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:50.1.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:50.1.0 editor:indent (atom-text-editor.editor.is-focused)
  6x -0:49.6.0 core:backspace (atom-text-editor.editor.is-focused)
  4x -0:47.7.0 core:undo (atom-text-editor.editor.is-focused)
     -0:39.1.0 editor:newline (atom-text-editor.editor.is-focused)
  5x -0:31.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:23.1.0 core:undo (atom-text-editor.editor.is-focused)
 14x -0:20.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:16.1.0 core:save (atom-text-editor.editor.is-focused)
     -0:11.3.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:09.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:09.2.0 coffeescript-check:check (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "change-case",
      "node-resolver",
      "npm-docs",
      "jsformat"
    ],
    "projectHome": "Users/robsawyer/Sites"
  }
}
```

### Installed Packages

```coffee
# User
atomatigit, v1.5.3
browser-plus, v0.0.25
coffeescript-check, v0.1.0
color-picker, v2.0.11
colorful-json, v1.0.0
jshint, v1.3.9
jsonlint, v1.1.2
minimap, v4.12.0
mocha, v0.0.7
node-debugger, v1.1.6
pigments, v0.9.0
pretty-json, v0.4.1
run-in-terminal, v0.4.1
terminal-panel, v1.14.1
travis-ci-status, v0.16.0

# Dev
No dev packages
```

**Comment by dogancelik** [comment]
I have this error too.

Windows 7 64-bit, Atom 1.0.7

This error happens when you select nothing and press <kbd>Ctrl+K</kbd> then <kbd>Ctrl+C</kbd>

**Comment by leny** [comment]
Hi, I'm *really* sorry for the long delay on this one.

Totally forgot this issue. 

I've just released version `0.2.0` that fixes this issue.
